test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml

```diff
@@ -18,7 +18,7 @@ instance_type_monitor: 't3.large'
 instance_type_db: 'i3en.2xlarge'
 
 nemesis_class_name: 'NemesisSequence'
-nemesis_interval: 60
+nemesis_interval: 30
```

Comment:
@roydahan @temichus @aleksbykov , changed it here, and it may affect somewhat latency during operations too, FYI

Reply:
I saw it, but how shall it affect it?
(And if it does, we need to remove the change)

Reply:
it will reduce the steady state... so my idea is now (next) to check the monitor of latest run, and confirm how long it does take for the "memory/cache" take to stabilize, during the steady state...
we have some time we wait beforehand, to heat the cache, so it should not affect at all, but need to be sure

Reply:
@fgelcer, did you check your idea?